File: TCC-App/src/app/models/Resultado/Resultado.ts
import { User } from '../User';
import { ExameResultado } from './ExameResultado';
import { PerguntaRespostaResultado } from './PerguntaRespostaResultado';

export class Resultado {
    id: number;
    dataHora: Date;
    alunoId: number;
    nomeAluno: string;
    professorId: number;
    nomeProfessor: string;
    hashLib: string;
    percAcertPergunta: number;
    percAcertExame: number;
    finalizado: boolean;
    nomePaciente: string;
    dataNascimento: Date;
    sexo: string;
    tipoAtendimento: string;
    queixaPrincipal: string;
    inicioSintomas: string;
    qtdMaxPergunta: number;    
    qtdMaxExame: number;    
    
    perguntaRespostasResultados: PerguntaRespostaResultado[];
    exameResultados: ExameResultado[];
}

File: TCC.Domain/resultados/Resultado.cs
using System;
using System.Collections.Generic;
using System.Text;
using TCC.Domain.Identity;

namespace TCC.Domain.resultados
{
    public class Resultado
    {
        public int Id { get; set; }
        public DateTime DataHora { get; set; }
        public int AlunoId { get; set; }
        public string NomeAluno { get; set; }
        public int ProfessorId { get; set; }
        public string NomeProfessor { get; set; }
        public decimal PercAcertPergunta { get; set; }
        public decimal PercAcertExame { get; set; }
        public bool Finalizado { get; set; }
        public string NomePaciente { get; set; }
        public string DataNascimento { get; set; }
        public string Sexo { get; set; }
        public string TipoAtendimento { get; set; }
        public string QueixaPrincipal { get; set; }
        public string InicioSintomas { get; set; }
        public int? QtdMaxPergunta { get; set; }
        public int? QtdMaxExame { get; set; }
        public string HashLib { get; set; }

        public List<PerguntaRespostaResultado> PerguntaRespostasResultados { get; set; }
        public List<ExameResultado> ExameResultados { get; set; }
    }
}
